pom.xml

```diff
@@ -181,11 +181,27 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>jackson2-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.pivovarit</groupId>
+            <artifactId>parallel-collectors</artifactId>
+            <version>2.6.1</version>
```

Comment:
This is up-to-date, but Renovate is trying to upgrade it to version 3.x in e.g. https://github.com/jenkinsci/junit-plugin/pull/629, which won't work because 3.x requires Java 21 or newer. Can we update the Renovate configuration to exclude 3.x until we are ready to require Java 21 or newer? That way, we will continue to get updates to version 2.x.